app/src/main/java/com/tehmou/rxbookapp/data/DataStoreModule.java

```diff
@@ -46,9 +47,15 @@ public ContentResolver contentResolver() {
     }
 
     @Provides
+    public Context applicationContext() {
+        return RxBookApp.getInstance().getApplicationContext();
```

Comment:
i believe that we should inject application via the constructor into the module rather than static reference